osbrain/proxy.py

```diff
@@ -174,7 +174,7 @@ def _is_safe_method(self, methodname):
         """
         return (methodname in self._pyroMethods and
                 not methodname.startswith('_') and
-                methodname not in ('ready', 'run', 'get_attr', 'kill',
+                methodname not in ('ping', 'run', 'get_attr', 'kill',
```

Comment:
Watching this I now remember why there were two different methods...

`ready` is expected to be a safe call (i.e.: it does not get serialized to the main thread). Putting `ping` in this line means there are some tests that may not be testing what they should.

I would say: remove `ping` form here. Then instead of replacing the `ready()`s with `ping()`s, try something like `.unsafe.ping()`.